app/components/project.tsx

```diff
@@ -0,0 +1,35 @@
+interface ProjectProps {
+    image?: string;
+    title?: string;
+    subtitle?: string;
+    description?: string;
+    link?: string;
+}
+
+const Project = ({
+    image,
+    title,
+    subtitle,
+    description,
+    link
+}: ProjectProps) => {
+    return (
+        <div className="flex flex-col items-center">
+            <img
+                src={image}
```

Comment:
for example:
```suggestion
                src={image ?? 'https://i.imgur.com/BlBH8Ak.jpeg' }
```
random panda picture for now